mappings/net/minecraft/client/render/item/ItemRenderer.mapping

```diff
@@ -4,20 +4,41 @@ CLASS net/minecraft/unmapped/C_xjqetogb net/minecraft/client/render/item/ItemRen
 	FIELD f_curtyzhw COMPASS_WITH_GLINT_GUI_MODEL_MULTIPLIER F
 	FIELD f_ddjgfxsn colors Lnet/minecraft/unmapped/C_bvzkhshe;
 	FIELD f_fbqbidyx ITEM_COUNT_BLIT_OFFSET I
+	FIELD f_kwmbfert ENCHANTED_GLINT_ITEM Lnet/minecraft/unmapped/C_ncpywfca;
```

Comment:
given the whole "natural word order" convention, I think we could move this to "ITEM_ENCHANTED_GLINT" since it's not a literal item
```suggestion
	FIELD f_kwmbfert ITEM_ENCHANTED_GLINT Lnet/minecraft/unmapped/C_ncpywfca;
```